### PR TITLE
Added the conferences page

### PR DESCRIPTION
**Pull request by digitsu**
Added conferences section and default page, with basic information on the Satoshi's Vision Conference

### CODE REVIEW

File: src/public/views/pages/conference.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+                        <h4 id="17e3" class="graf--h4 graf-after--p">Acknowledgement</h4>
+                        <p id="5030" class="graf--p graf-after--h4 graf--last">We gratefully acknowledge the extremely generous donations made anonymously to the <a class="markup--anchor markup--p-anchor" href="http://bitcoinunlimited.info/" target="_blank" rel="nofollow" data-href="http://bitcoinunlimited.info">Bitcoin Unlimited Community Organization</a> that make this event possible.</p>
```
Comment (thofmann) [inline src/public/views/pages/conference.jsx]: This link should be `https`, but the user will still be redirected from `http` to `https` anyway. Not a blocking issue.

File: src/public/views/pages/conference.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+export default React.createClass({
+
+    render: function() {
+        return (
+            <div id='conference'>
+                <Header active='conference' />
+                <Title title="Satoshi's Vision" />
+                <div className='section'>
+                    <div className='container'>
+                        <h1 id="5eed" class="graf--h3 graf--leading graf--title">Satoshi’s Vision: Bitcoin Development &amp; Scaling Conference</h1>
```
Comment (thofmann) [inline src/public/views/pages/conference.jsx]: `class` should be `className` here and in other places, right?

File: src/public/views/pages/conference.jsx (hunk 1 above)
Comment (digitsu) [inline src/public/views/pages/conference.jsx]: thanks.  Assume you mean the one to bitcoinunlimited.info.


Good catch for future reference.

**Comment by thofmann** [comment]
Concept ACK, will test tomorrow.

**Comment by thofmann** [comment]
Looks great! Thanks for the PR!

I suggested a few minor updates here: https://github.com/digitsu/BitcoinUnlimitedWeb/pull/1

If you wouldn't mind merging those updates, I'm prepared to merge and deploy this PR.

**Comment by digitsu** [comment]
merged up your changes thofmann

**Comment by digitsu** [comment]
Merged and re submitted.

**Comment by thofmann** [comment]
:+1:

**Pull request merged**